central/networkgraph/service/service_impl_test.go

```diff
@@ -355,11 +355,11 @@ func (s *NetworkGraphServiceTestSuite) TestGenerateNetworkGraphWithSAC() {
 				testutils.DepFlow("depA", "depX"),
 				testutils.DepFlow("depA", "depY"),
 				testutils.DepFlow("depA", "depZ"),
-				testutils.AnyFlow("depA", storage.NetworkEntityInfo_DEPLOYMENT, es1aID.String(), storage.NetworkEntityInfo_EXTERNAL_SOURCE),
-				testutils.AnyFlow("depA", storage.NetworkEntityInfo_DEPLOYMENT, es1bID.String(), storage.NetworkEntityInfo_EXTERNAL_SOURCE),
-				testutils.AnyFlow("depA", storage.NetworkEntityInfo_DEPLOYMENT, es1cID.String(), storage.NetworkEntityInfo_EXTERNAL_SOURCE),
-				testutils.AnyFlow("depA", storage.NetworkEntityInfo_DEPLOYMENT, es2ID.String(), storage.NetworkEntityInfo_EXTERNAL_SOURCE),
-				testutils.AnyFlow("depA", storage.NetworkEntityInfo_DEPLOYMENT, es5ID.String(), storage.NetworkEntityInfo_EXTERNAL_SOURCE),
+				testutils.AnyFlow("depA", storage.NetworkEntityInfo_DEPLOYMENT, es1aID.String(), storage.NetworkEntityInfo_EXTERNAL_SOURCE, ""),
```

Comment:
I have added clusterID as a parameter to AnyFlow. Where AnyFlow was called before, I now just pass in an empty string.

Reply:
Yes, the empty string is the value representing the global scope

Reply:
That caused a lot of unnecessary noise.  Why not simply create a new function called `AnyFlowWithCluster` and use that where needed.  That would probably read better and would have eliminated 100s of lines of change.

Reply:
I considered doing that, but having `AnyFlow` and `AnyFlowWithCluster` seems like unnecessary code duplication. It would be nice if go allowed functions with default values for arguments.